multiapi-engine/src/main/java/com/sngular/api/generator/plugin/asyncapi/model/ProcessBindingsResult.java

```diff
@@ -0,0 +1,24 @@
+/*
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  * License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.sngular.api.generator.plugin.asyncapi.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ProcessBindingsResult {
+
+  private String bindings;
+
+  private String bindingType;
```

Comment:
Could have sense let as the `BindingTypeEnum` instead of a String which is always forced to have the value of the Enum?

Reply:
Variable names should have business meaning not related to how is implemented. Exception List and Maps postfix to indicate is a collection and only in cases where you are allowing to add single elements.